tests/cases/compute_nothing.amber

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[compute shader]
+#version 430
+
+void main() {
+}
+
+[test]
+compute 4 1 1
```

Comment:
Do we have a compute shader test which does do something?

Reply:
The problem here is that we cannot check the result because probe ssbo command is not implemented yet.
Actually I already implemented probe ssbo command and checked a simple add/minus/multiplication working.
I will send another PR for the probe ssbo command after this one is merged.